pycbc/io/live.py

```diff
@@ -506,6 +499,8 @@ def upload(self, fname, gracedb_server=None, testing=True,
                          if v != 0.0}
             labels, sizes = zip(*prob_plot.items())
             colors = [source_color(label) for label in labels]
+            import pylab as pl
```

Comment:
As codeclimate noted, this is not needed, pylab is imported at the beginning of this (insanely long) method.